### PR TITLE
Developed Team Meeting Plan

### PR DESCRIPTION
**Pull request by harshc22**
- stated the frequency of meetings, meeting locations, meeting structure, meeting protocols

- compiled the pdf for development plan

- updated common.tex to contain author names and team name

- testing the branch rules (at least one review before merge)

### CODE REVIEW

**Review by martinivnv — DISMISSED** [review]

**Comment by martinivnv** [comment]
Don't forget to document these revisions in the revision history

**Comment by harshc22** [comment]
- Added the expected technology and coding standard
- Added the revision in the revision history

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 1]
```diff
@@ -143,42 +143,42 @@ \section{Proof of Concept Demonstration Plan}
...
+    The project will create a \textbf{custom AI model} specifically tailored to dairy farming data, focusing on predicting outcomes such as breeding success rates and herd attrition.
+    \item \textbf{Linter}: 
+    \textbf{Flake8} or \textbf{Black} will be utilized to maintain code quality and ensure adherence to Python's PEP 8 standards.
+    \item \textbf{Unit testing framework}: 
+    \textbf{PyTest} will be used to implement unit tests, focusing on validating data pipelines, model training, and prediction accuracy.
+    \item \textbf{Continuous Integration (CI)}: 
+    \textbf{GitHub Actions} will be used to automate tests and ensure code quality in a continuous integration pipeline.
+    \item \textbf{Version Control and Project Management}: 
+    \textbf{Git} will be used for version control and \textbf{GitHub} for repository management and collaboration throughout the project.
+\end{itemize}
```
Comment (ShazimR) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:163]: I guess we can add more to this later as we define the requirements better.

**Review by ShazimR — APPROVED** [review]
Otherwise looks good to me!

File: docs/DevelopmentPlan/DevelopmentPlan.tex (hunk 1 above)
Comment (harshc22) [reply docs/DevelopmentPlan/DevelopmentPlan.tex:163]: Yeah, for sure

**Pull request merged**